ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/util/SampleData.java

```diff
@@ -0,0 +1,63 @@
+package uk.gov.hmcts.cmc.ccd.util;
```

Comment:
we got two SampleData class, please merge into one.

Reply:
done